app/models.py

```diff
@@ -51,29 +58,48 @@ def as_dict(self):
 class Election(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
-    slug = db.Column(db.String())
-    user_id = db.Column(db.Integer, db.ForeignKey('user.id'))
     name = db.Column(db.String(), nullable=False, index=True)
     description = db.Column(db.String())
     created_at = db.Column(db.DateTime, default=datetime.utcnow)
     modified_at = db.Column(db.DateTime, default=datetime.utcnow)
     starting_at = db.Column(db.DateTime, index=True)
     ending_at = db.Column(db.DateTime)
     link = db.Column(db.String())
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'))
     status_id = db.Column(db.Integer, db.ForeignKey(
         'status.id'), nullable=False)
+    positions = db.relationship(
+        'Position', backref='election', lazy='dynamic',
+        passive_deletes=True, cascade="all, delete")
     number_of_voters = db.Column(db.String(), nullable=False)
     password = db.Column(db.String())
+
+    def __repr__(self):
+        return f'<Election {self.name}>'
+
+    def as_dict(self):
+        return {item.name: getattr(self, item.name) for item in self.__table__.columns}
+
+
+class Position(db.Model):
+
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(), nullable=False, index=True)
+    description = db.Column(db.String())
+    created_at = db.Column(db.DateTime, default=datetime.utcnow)
+    modified_at = db.Column(db.DateTime, default=datetime.utcnow)
+    election_id = db.Column(db.Integer, db.ForeignKey(
+        'election.id', ondelete='CASCADE'), nullable=False)
     candidates = db.relationship(
-        'Candidate', backref='election', lazy='dynamic',
+        'Candidate', backref='position', lazy='dynamic',
```

Comment:
are the candidates, votes and results fields all meant to have backref of position

Reply:
There is a relationship between Candidates and Position. So to see the Position a Candidate belongs to you just do `Candidate.Position`. There is also a relationship between Result and Position, Vote and Position. Whether these relationships should even exist the way they are is another question.